libraries/helpers.rb

```diff
@@ -20,6 +20,13 @@ def fs_type(mount)
     def fs_remote?(mount)
       fs_type(mount) == 'nfs' ? true : false
     end
+
+    # Validates passed id.
+    #
+    # @return [Numeric, String]
+    def validate_id(id)
+      id.to_i.to_s == id ? id.to_i : id
```

Comment:
On 1/21/16, Tim Smith <notifications@github.com> wrote:
>> @@ -20,6 +20,13 @@ def fs_type(mount)
>>      def fs_remote?(mount)
>>        fs_type(mount) == 'nfs' ? true : false
>>      end
>> +
>> +    # Validates passed id.
>> +    #
>> +    # @return [Numeric, String]
>> +    def validate_id(id)
>> +      id.to_i.to_s == id ? id.to_i : id
>
> @375gnu Is there any reason not to just make this 'id.to_i'?  If the ID is a
> string this will return the int and if it's an int it'll return the int.  It
> avoids additional computation on the value with each chef run.

Because #to_i retuns 0 if value can't be converted to int. I don't
think that you want .ssh to be owned by root.